services/web-app/components/catalog-multislect-filter/catalog-multislect-filter.js

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright IBM Corp. 2021, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { Column, Tag } from '@carbon/react'
+import { ChevronDown, ChevronUp, OverflowMenuHorizontal } from '@carbon/react/icons'
+import { useRef, useState } from 'react'
+
+import { useOutsideClick } from '@/utils/use-outside-click'
```

Comment:
Not sure if custom outside click events will be necessary if you use ListBox components to make this custom dropdown filter.

https://github.com/carbon-design-system/carbon/tree/main/packages/react/src/components/ListBox